src/anm/media/video.js

```diff
@@ -47,7 +47,7 @@ Video.prototype.load = function(elm, player) {
     var me = this;
     ResMan.loadOrGet(player.id, me.url,
         function(notify_success, notify_error, notify_progress) { // loader
-            var url = me.url;
+            var url = Array.isArray(me.url) && me.url.length > 0 ? me.url[0] : me.url;
```

Comment:
sorry, it was my fault. somehow this array crept in, funny thing I remember that a test failed, but I just went and "fixed" it by supplying wrong data thinking it's correct.

I will fix it on editor side, so no need for this PR